packages/valory/agents/apy_estimation/aea-config.yaml

```diff
@@ -64,7 +64,7 @@ dependencies:
   open-aea-ledger-cosmos: {}
   open-aea-ledger-ethereum: {}
   open-aea-test-autonomy:
-    version: ==0.3.2
```

Comment:
This was causing the e2e failures. We haven't published the v0.3.2 on the PYPI yet.

Reply:
Yes - but that shouldn’t be done like this.

1) our CI should never actually require installing the package from PyPI. We need to use the local installation as this package is developed here too. We already do so but some code is wrongly trying to install the deps, which is not what we want in CI of this repo.

2) the release obviously needs to bump to the new versions, so this latest commit makes no sense.